docs/release_notes.rst

```diff
@@ -4,6 +4,15 @@ Release notes
 develop
 -------
 
+New dependencies
+^^^^^^^^^^^^^^^^
+
+* This version of Bodhi now requires pungi >= 4.1.18.
+* As pungi 4 is not available in EPEL 7, this release of Bodhi formally drops support for EL 7. The
+  supported operating systems only include active Fedora releases now, and are documented on Bodhi's
+  docs' front page.
+
```

Comment:
https://koji.fedoraproject.org/koji/buildinfo?buildID=978281 has pungi for epel7

Reply:
True, we should correct the language here and just say that EL 7 is no longer supported. I guess we don't *have* to explain why.